scripts/find_and_tag_cdc_stable_candidate.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Convenience script for finding and tagging a custom DC service image that
+# corresponds to the latest website prod release. Note that this will change
+# which image the stable-candidate tag applies to.
+
+# Usage: ./scripts/find_and_tag_cdc_stable_candidate.sh
+
+set -e
+
+###### BEGIN CONFIG ######
+
+# Tag of the website commit that we'll attempt to find a matching CDC image for.
+WEBSITE_PROD_TAG="prod"
```

Comment:
Can the script take the website tag as a parameter? Also, does it need to be a tag or can we get it to work with a commit sha as well?

Reply:
Sure, both are doable, though it's more work and maintenance to have the script be able to do more things. What will the standard workflow be?

Reply:
The tag would either be based on a prod tag or on a specific commit. It'd be good to support both.

Reply:
Ok, working on adding this now.